core/src/main/java/org/stellar/anchor/sep38/Sep38Service.java

```diff
@@ -507,4 +524,9 @@ private String getTotalPrice(BigDecimal bSellAmount, BigDecimal bBuyAmount) {
 
     return formatAmount(bTotalPrice, pricePrecision);
   }
+
+  private String getClientIdFromToken(Sep10Jwt token) {
+    if (token == null) return null;
+    return token.getAccount();
+  }
```

Comment:
This works until we add support for using SEP-38 prices & quotes for SEP-24 & SEP-6 transactions. If we want to replace the usage of public keys and domains with client names, we can't merge this because it would be a breaking change to change `clientId` from a public key to names defined in the config. cc @Ifropc